### ISSUE TITLE
Unable to select SR to build VM on when using knife xapi guest create 

### ISSUE THREAD

**Issue by Taurino3** [spheromak/knife-xapi#12]
I have a pool of xenservers, there is one master and 2 slaves.

I want to be able to build a VM on a specific host in the pool even though the host is a slave vs a pool master. 

I figured if I use the --xapi-sr command I would be able to select the slave host local storage SR and then the VM would be created on that host....

I cannot find an example anywhere of how to use the --xapi-sr command...

Basically what i am doing now is the below not sure that is correct...

--xapi-sr {SR UUID} --host http://{pool master}
